test/integration/specs/kbaseSampleSetViewer_test.js

```diff
@@ -0,0 +1,79 @@
+/*global describe, it, browser, expect, $, afterEach, beforeEach*/
+/* eslint {strict: ['error', 'global']} */
+'use strict';
+
+const {login, openNarrative} = require('../wdioUtils.js');
+
+describe('Test kbaseNarrativeSidePublicTab', () => {
+    beforeEach(async () => {
+        await browser.setTimeout({ 'implicit': 30000 });
+        await browser.reloadSession();
+        await login();
+        await openNarrative(58574);
```

Comment:
is this specific to a certain environment - e.g. CI? Erik added some code to his recent PR to pick the appropriate narrative ID according to the env: https://github.com/kbase/narrative/pull/2076/files#diff-f67c5ffeab78699f76db06df181b1d5861b50f650e0b25c14c44c39a811d59f5. That might be appropriate here, or, if the narrative is only in one environment, you could check that you're in the appropriate env before running the tests.

Reply:
My understanding is that integration tests run against CI, though I did notice that in Erik's code. I'll make a note to check in with him about it when he's back

Reply:
Though, looks like the integration test action failed, so maybe that's the reason. looking into it.

Reply:
Tests seem to be particularly flaky at the moment -- although they will often pass if you rerun them.

Reply:
Erik is creating a more solid framework for integration tests in #2076, so it might be better to wait until that PR is integrated so you can use the functionality he's created.